metadata.rb

```diff
@@ -17,4 +17,4 @@
 supports 'scientific'
 supports 'oracle'
 
-depends 'yum-epel'
+depends 'yum-epel', '~> 3.3.0'
```

Comment:
This should be a >= for a community cookbook. Constraining deps creates impossible depsolving issues over time.